src/Map/src/Bridge/Leaflet/assets/yarn.lock

```diff
@@ -0,0 +1,49 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Must it be commited ?  If yes, can we add it to .gitattributes ? 

(same for both bridges)